GoogleForCreators/web-stories-wp
# Remove Text element quick-add button

## Context

This is partial PR for the Insertion logic changes of #10109 


## Summary
Removes Text quick-add action from the tabs and updates all the relevant tests.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Verify the Library Text tab doesn't have the quick-add button anymore.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Partially addresses #10109
